src/components/details/SideView.js

```diff
@@ -0,0 +1,74 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faArrowCircleRight, faGear } from '@fortawesome/free-solid-svg-icons';
+import { Link } from 'react-router-dom';
+
+const SideView = (props) => {
+  const { motorcycle } = props;
+
+  const deposit = motorcycle.price * 0.05;
+
+  const serviceFee = motorcycle.price * 0.01;
+
+  const financeFee = motorcycle.price * 0.03;
+
```

Comment:
Good job calculating deposit, service fee and finance fee in the script without adding them in the db